railties/test/application/configuration_test.rb

```diff
@@ -3166,6 +3166,23 @@ class MyLogger < ::Logger
       assert_includes Rails.application.config.hosts, ".localhost"
     end
 
+    test "ActiveStorage.transformations_for_format is an empty hash by default" do
+      app "development"
+      assert_equal Hash.new, ActiveStorage.transformations_for_format
+    end
+
+    test "ActiveStorage.transformations_for_format can be configured via config.active_storage.transformations_for_format" do
+      app_file "config/environments/development.rb", <<-RUBY
+        Rails.application.configure do
+          config.active_storage.transformations_for_format[:jpg] = { format: "jpg" }
+        end
+      RUBY
+
+      app "development"
+
+      assert({ format: "jpg" }, ActiveStorage.transformations_for_format[:jpg])
```

Comment:
We gotta fix this assertion or the test would always pass:

```suggestion
      assert_equal({ format: "jpg" }, ActiveStorage.transformations_for_format[:jpg])
```